project_euler/problem_024/sol2.py

```diff
@@ -0,0 +1,35 @@
+"""A permutation is an ordered arrangement of objects. For example, 3124 is one possible permutation of the digits 1, 2, 3 and 4. If all of the permutations are listed numerically or alphabetically, we call it lexicographic order. The lexicographic permutations of 0, 1 and 2 are:
```

Comment:
Please split this line:
`project_euler/problem_024/sol2.py:1:89: E501 line too long (280 > 88 characters)`